app/controllers/shopify_app/sessions_controller.rb

```diff
@@ -27,6 +27,7 @@ def top_level_interaction
     def destroy
       reset_session
       flash[:notice] = I18n.t(".logged_out")
+      Rails.logger.debug("ShopifyApp - Destroy Session and redirecting to login")
```

Comment:
I wonder if we can use Tagged Logging so we don't need to repeat `ShopifyApp` everywhere: https://api.rubyonrails.org/classes/ActiveSupport/TaggedLogging.html